docs/api/covidcast_changelog.md

```diff
@@ -0,0 +1,28 @@
+# Delphi's COVID-19 Surveillance Streams Changelog
+
+When Delphi makes substantial changes to the computation of a signal, we will typically publish it under a new name rather than revising the existing signal. By contrast, the changes documented here are bug fixes and pipeline repairs.
+
+## Sources and Signals
+### `doctor-visits`
+### `fb-survey`
+
+<a name="fb-survey-v1.3"></a> 
+#### 3 June 2020: v1.3
```

Comment:
In theory, this heading already has an anchor with id `3-june-2020-v1.3` or something like that.

I think, though, that it would be easier for each source section in `covidcast_signals.md` to link to the corresponding section in `covidcast_changelog.md`, rather than the section for the most recent update. So you'd only have to link to `covidcast_changelog.md#fb-survey`, where the user would immediately see the latest changes. That would make updates easier.

Reply:
Done.